ultralytics/engine/exporter.py

```diff
@@ -122,6 +124,7 @@ def export_formats():
         ["MNN", "mnn", ".mnn", True, True, ["batch", "half", "int8"]],
         ["NCNN", "ncnn", "_ncnn_model", True, True, ["batch", "half"]],
         ["IMX", "imx", "_imx_model", True, True, ["int8"]],
+        ["RKNN", "rknn", "_rknn_model", False, False, ["name"]],
```

Comment:
@lakshanthad @ambitious-octopus all export formats need to pass all available arguments here. The docs page says it supports the "batch" argument too?

Reply:
Fixed!